### PR TITLE
Test metadata propagation through aliases

### PR DESCRIPTION
**Pull request by tomasfejfar**
Tests https://github.com/keboola/connection/pull/2144

### CODE REVIEW

File: tests/Common/MetadataTest.php [hunk 1]
```diff
@@ -237,6 +237,50 @@ public function testColumnMetadata()
...
+            $this->getTestBucketId(),
+            $this->getTestBucketId() . '.tableAlias',
+            'tableAliasAlias'
+        );
+
+        // test list tables call
+        $tables = $this->_client->listTables(null, ['include' => 'columnMetadata']);
+        // call return all tables, filter the alias of alias one
+
+        $testedTableId = 'in.c-API-tests.tableAliasAlias';
```
Comment (pivnicek) [inline tests/Common/MetadataTest.php]: better to use `$this->getTestBucketId()` here imho

File: tests/Common/MetadataTest.php [hunk 2]
```diff
@@ -237,6 +237,50 @@ public function testColumnMetadata()
...
+        );
+
+        $tables = $this->_client->listTables($this->getTestBucketId(), ['include' => 'columnMetadata']);
+        $tables = array_values(array_filter($tables, function ($table) use ($testedTableId) {
+            return $table['id'] === $testedTableId;
+        }));
+        $this->assertEquals(
+            $mdList,
+            $tables[0]['sourceTable']['columnMetadata']['id']
+        );
```
Comment (pivnicek) [inline tests/Common/MetadataTest.php]: Is this not identical to the assertion on line 264?

**Comment by tomasfejfar** [comment]
Fixed

**Comment by tomasfejfar** [comment]
Added test for linked bucket alias metadata propagation.

**Pull request merged**